addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp

```diff
@@ -0,0 +1,572 @@
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Rangefinder",
+		"ItemGPS"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	uniform[] = {"U_B_FullGhillie_sard"};
+	vest[] = {"V_Chestrig_rgr"};
+	backpack[] = {"B_AssaultPack_cbr"};
+	headgear[] = {"CUP_H_OpsCore_Covered_MTP_NoHS"};
+	primaryWeapon[] = {"CUP_srifle_AWM_wdl"};
+	scope[] = {"cup_optic_leupoldmk4_20x40_lrt_pip"};
+	bipod[] = {"cup_bipod_harris_1a2_l_blk"};
+	sidearmWeapon[] = {"CUP_hgun_MP7"};
+	magazines[] = {
+		LIST_7("CUP_5Rnd_86x70_L115A1"),
+		LIST_2("HandGrenade"),
+		LIST_4("CUP_40Rnd_46x30_MP7")
+	};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	uniform[] = {"U_B_FullGhillie_sard"};
+	vest[] = {"V_Chestrig_rgr"};
+	silencer[] = {"cup_muzzle_snds_m16"};
+	items[] += {
+		"ACE_MapTools",
+		"ACE_RangeCard",
+		"ACE_Kestrel4500",
+		"ACE_ATragMX"
+	};
+	backpack[] = {"B_AssaultPack_cbr"};
+	backpackItems[] = {
+		LIST_6("CUP_5Rnd_86x70_L115A1")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Laserdesignator",
+		"ItemGPS"
+	};
+};
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	vest[] = {"V_Rangemaster_belt"};
+	backpack[] = {};
+	headgear[] = {"CUP_H_USMC_ACVC_DES"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class vc : vg {
+	displayName = "Vehicle Commander";
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class vd : vg {
+	displayName = "Vehicle Driver";
+	traits[] = {"engineer"};
+	backpack[] = {"B_AssaultPack_mcamo"};
+	linkedItems[] += {"ItemGPS"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class hc : smg {
+	displayName = "Helicopter Crew";
+	uniform[] = {"MNP_CombatUniform_Australia"};
+	vest[] = {"FIR_pilot_vest"};
+	backpack[] = {};
+	headgear[] = {
+		"CFP_SOARCREW_BLUE",
+		"CFP_SOARCREW_STACHE",
+		"CFP_SOARCREW_RED",
+		"CFP_SOARCREW_SKULL",
+		"H_CrewHelmetHeli_B",
+		"FIR_HGU56P_Skull",
+		"FIR_HGU56P_Skull_Death",
+		"MNP_Helmet_Dslyecxi"
+	};
+	items[] += {"ACE_MapTools"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class hp : hc {
+	displayName = "Helicopter Pilot";
+	uniform[] = {"FIR_Fighter_Pilot_Uniform_Flight"};
+	linkedItems[] += {"ItemGPS"};
+};
+
+class hcc : hc {
+	displayName = "Helicopter Crew Chief";
+	traits[] = {"engineer"};
+	backpack[] = {"B_AssaultPack_mcamo"};
+	linkedItems[] += {"ItemGPS"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class jp : smg {
+	displayName = "Jet pilot";
+	headgear[] = {"FIR_JHMCS"};
+	hmd[] = {};
+	vest[] = {"FIR_pilot_vest"};
+	traits[] = {"engineer"};
+	uniform[] = {"FIR_Fighter_Pilot_Uniform_Flight"};
+	backpack[] = {};
+	linkedItems[] += {"ItemGPS"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class eng : car {
```

Comment:
Add ACE mine detector

Reply:
The change request is showing me the Jet Pilot kit; are they supposed to have an ACE Mine Detector, ala VMM_3 or VMH_3?

Reply:
Comment was posted on ENG/ENGM. Yeah, add either VMM or VMH.